### PR TITLE
Documentation update and cleanup

### PR DESCRIPTION
**Pull request by pfistererm**
- updated documentation
- POM cleanup
- added integration test build to travis
- removed obsolete launch configurations
- integration test cleanup
- minor refactoring
- updated snapshot version number

## Type of request

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Refactoring 
- [x] Documentation or documentation changes

#### Initiator
Matthias Pfisterer

## Checklist:

- [x] All the commits are signed.
- [x] My code follows the code style of this project.
- [x] I agree with die CLA.
- [x] I have read the CONTRIBUTING docs.
- [x] I have added/updated necessary documentation (if appropriate).

### CODE REVIEW

**Comment by pfistererm** [comment]
Once this PR is merged, also #10 can finally be closed, since now integration tests are also executed in the travis build.

File: licensescout-documentation/src/site/asciidoc/cookbook/cookbook.adoc [hunk 1]
```diff
@@ -40,7 +40,23 @@ TODO
 TODO
 
 
+== Documenting manually researched Licenses
 
+Once you have researched Licenses of an artifact manually, it is also
+necessary to document the findings in a structured way.
+It may be after years that some legal department is asking why you came to the idea that
+a certain artifact has a certain license. So it is euired to:
```
Comment (davu-all) [inline licensescout-documentation/src/site/asciidoc/cookbook/cookbook.adoc]: typo on "euired" (?)

File: licensescout-documentation/src/site/asciidoc/maintenance/maintenance.adoc [hunk 2]
```diff
@@ -25,6 +25,19 @@
 
 This handbook describes project maintenance tasks for the LicenseScout project.
 
+== General conventions
+
+* All files in the repository should be encoded in `UTF-8`. Exceptions are test resource files that are used to test other encodings.
+* All files in the repository should have Unix-style lline endings (LF).
```
Comment (davu-all) [inline licensescout-documentation/src/site/asciidoc/maintenance/maintenance.adoc]: typo on "lline"

File: licensescout-parent/pom.xml [hunk 3]
```diff
@@ -170,7 +170,7 @@
 		<maven.compiler.release>${java.version}</maven.compiler.release>
 
 		<!-- Github upload/publish -->
-		<github.owner>aposin</github.owner>
```
Comment (davu-all) [inline licensescout-parent/pom.xml]: shouldn't this remains "aposin"?

Comment (pfistererm) [reply licensescout-parent/pom.xml]: Yes. I changed this back.

File: licensescout-documentation/src/site/asciidoc/maintenance/maintenance.adoc (hunk 2 above)
Comment (pfistererm) [reply licensescout-documentation/src/site/asciidoc/maintenance/maintenance.adoc]: Fixed.

File: licensescout-documentation/src/site/asciidoc/cookbook/cookbook.adoc (hunk 1 above)
Comment (pfistererm) [reply licensescout-documentation/src/site/asciidoc/cookbook/cookbook.adoc]: Fixed.

File: licensescout-core/src/main/java/org/aposin/licensescout/configuration/ConfigFileHandlerHelper.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+ * limitations under the License.
+ */
+package org.aposin.licensescout.configuration;
+
+import java.io.File;
+
+import org.aposin.licensescout.util.ILFLog;
+
+/**
+ * Helper class for instantiating ConigFileHandler instances.
```
[the remark this answers is not in the document]
Comment (pfistererm) [reply licensescout-core/src/main/java/org/aposin/licensescout/configuration/ConfigFileHandlerHelper.java]: Fixed.

File: licensescout-core/pom.xml [hunk 5]
```diff
@@ -7,21 +7,17 @@
...
+	<version>1.4.${revision}</version>
 	<packaging>jar</packaging>
 
 	<name>LicenseScout Core</name>
-	<description>Finds license information and collects them to a license report.</description>
-
-	<prerequisites>
-		<maven>3.3.0</maven>
-	</prerequisites>
+	<description>Processing core of the LicenseScout that does read configuration files, scan for licenses and rites reports.</description>
```
[the remark this answers is not in the document]
Comment (pfistererm) [reply licensescout-core/pom.xml]: Fixed.

**Pull request merged**